### PR TITLE
Electron-325 (Removed restriction from opening a popup from another popup window)

### PR DESCRIPTION
**Pull request by KiranNiranjan**
## Description
Removed restriction from opening a popup from another popup window [ELECTRON-325](https://perzoinc.atlassian.net/browse/ELECTRON-325)

## Approach
How does this change address the problem?
#### Problem with the code: 
- The previous implementation was restricted to popup a window only from the main window.
#### Fix:
- Removed the restriction which validates and creates the pop-out window

## Before
electron-325-before

## After
electron-325-after

## Spectron test results
[Test Results — Spectron.pdf](https://github.com/symphonyoss/SymphonyElectron/files/1796756/Test.Results.Spectron.pdf)

## Unit test results
[Test Results — Unit Tests.pdf](https://github.com/symphonyoss/SymphonyElectron/files/1796758/Test.Results.Unit.Tests.pdf)

## Learning
- N/A

#### Blog Posts
- N/A

## Open Questions if any and Todos
- [X] Unit-Tests (N/A)
- [x] Documentation
- [X] Automation-Tests
When solved, check the box and explain the answer.

@VikasShashidhar & @VishwasShashidhar Please review. Thanks 😄

### CODE REVIEW

File: demo/win.html [hunk 1]
```diff
@@ -1,12 +1,24 @@
...
 
+<button id="open-win">Open a new child window</button>
+<br>
+<br>
+
+<button id="open-in-browser">Open Symphony in browser</button>
+<br>
+<br>
+
+<a href="https://symphony.com" target="_blank">Open Symphony Anchor Tag</a>
```
Comment (VikasShashidhar) [inline demo/win.html]: `Symphony Anchor Tag`? What is that supposed to mean?

File: tests/spectron/popOuts.spectron.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+    let originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = Application.getTimeOut();
+
+    beforeAll((done) => {
+        return app.startApplication().then((startedApp) => {
+            app = startedApp;
+            done();
+        }).catch((err) => {
+            console.error(`Unable to start application error: ${err}`);
```
Comment (VikasShashidhar) [inline tests/spectron/popOuts.spectron.js]: Could simply be `Unable to start application: ${err}`.

File: tests/spectron/popOuts.spectron.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+                done();
+            }).catch((err) => {
+                expect(err).toBeNull();
+            });
+        }).catch((err) => {
+            expect(err).toBeNull();
+        });
+    });
+
+    it('should check window count', () => {
```
Comment (VikasShashidhar) [inline tests/spectron/popOuts.spectron.js]: What is this test supposed to do?

File: tests/spectron/popOuts.spectron.js (hunk 2 above)
Comment (VishwasShashidhar) [reply tests/spectron/popOuts.spectron.js]: agreed

**Comment by VishwasShashidhar** [comment]
@KiranNiranjan Have you made the changes that @VikasShashidhar has requested? If so, please let me know and I'll merge this PR.

**Comment by KiranNiranjan** [comment]
@VishwasShashidhar & @VikasShashidhar I've made all the required changes. Please review. Thanks 😄

**Comment by VishwasShashidhar** [comment]
@VikasShashidhar Please approve so I can merge the changes.

**Pull request merged**